pathTOYellowMask : '/home/mscv_gr1/catkin_ws/src/MSCV-Robotics-Project/turtlebot3_autorace_lane/config/mask_yellow.png'
pathTOWhiteMask : '/home/mscv_gr1/catkin_ws/src/MSCV-Robotics-Project/turtlebot3_autorace_lane/config/mask_white.png'


YellowMaskMinHSV : 
  - 0
  - 100
  - 170

YellowMaskMaxHSV :
  - 90
  - 255
  - 255

WhiteMaskMinHSV :
  - 30
  - 0
  - 173

WhiteMaskMaxHSV :
  - 90
  - 65
  - 250


top_x1 : 0
top_y1 : 415
bottom_x1 : 640
bottom_y1 : 415


top_x2 : 0 
top_y2 : 480
bottom_x2 : 640
bottom_y2 : 480


contrast : 20
sharpness : 100
brightness : 50
saturation : 25
ISO : 100
exposure_compensation : -4